src/main.py

```diff
@@ -263,6 +263,9 @@ def __init__(self, aconfig):
 
 		self._aconfig = aconfig
 
+		if not self._aconfig['refresh_token']:
```

Comment:
Okay. How about 

```python
logger.critical("AVS refresh_token not found in the configuration file. Run the setup again to fix your installation (see project wiki for installation instructions).")
sys.exit()
```

I could say the auth_web script instead of setup, but it's probable that when the token is missing, setup wasn't run at all or got messed up. Is that okay?